peregrine/gps_constants.py

```diff
@@ -24,3 +24,5 @@
 
 L1CA = 'l1ca'
 L2C = 'l2c'
+
+L2C_CAPB = 0xf7814bfd
```

Comment:
Let's a comment describing the constant and the reference to a place where the latest status could be looked-up. For example, https://www.unavco.org/projects/project-support/gnss-support/gnss-modernization/gnss-modernization.html